src/components/now-playing-drawer/PlayBar.tsx

```diff
@@ -103,6 +99,8 @@ const createStyles = (themeColors: ThemeColors) =>
   })
 
 type PlayBarProps = {
+  track: Track | null
```

Comment:
Can you null check at the parent level so that the PlayBar doesn't render when track or user is null? Or is that dangerous

Reply:
It should be fine to do that b/c the drawer doesn't show at all when there's nothing playing. Updated!